examples/requirements.txt

```diff
@@ -1,4 +1,5 @@
 transformers~=4.27.1
 diffusers==0.14.0
 scipy~=1.10.0
-accelerate==0.17.1
\ No newline at end of file
+accelerate==0.17.1
+redis
```

Comment:
Actually, if `redis` is added to `pyproject.toml` (which it needs to be), this should just be deleted from `examples/requirements.txt`, since this only lists supplementary requirements beyond those needed for the package.
```suggestion
```